src/main.rs

```diff
@@ -42,29 +49,39 @@ fn main() -> Result<()> {
     .expect("No interactive terminal");
 
     let matches = public_clap_app()
-        .subcommand(
-            SubCommand::with_name("init")
-            .about("Prints the shell function used to execute emplace")
-            .arg(
-                Arg::with_name("shell")
-                .value_name("SHELL")
-                .help(
-                    "The name of the currently running shell\nCurrently supported options: bash & zsh",
-                )
-                .required(true)
-            )
-        )
-        .subcommand(
-            SubCommand::with_name("catch")
-                .about("Capture a command entired in a terminal")
-                .arg(
-                    Arg::with_name("line")
-                        .value_name("LINE")
-                        .help("The command as entired in the terminal")
-                        .required(true),
-                ),
-        )
-        .get_matches();
+		.subcommand(
+			SubCommand::with_name("init")
+				.about("Prints the shell function used to execute emplace")
+				.arg(
+					Arg::with_name("shell")
+						.value_name("SHELL")
+						.help(
+							"The name of the currently running shell\nCurrently supported options: bash & zsh",
+						)
+						.required(true)
+				)
+		)
+		.subcommand(
+			SubCommand::with_name("catch")
+				.about("Capture a command entered in a terminal")
+				.arg(
+					Arg::with_name("line")
+						.value_name("LINE")
+						.help("The command as entired in the terminal")
+						.required(true),
+				),
+		)
+		.subcommand(
+			SubCommand::with_name("history")
+				.about("Parses your history file and retrieves installations")
+				.arg(
+					Arg::with_name("history_parse")
+						.help("Parses history. Just place `$HISTFILE` as input, and it will do all work;")
+						.required(false)
+						.takes_value(true)
+				),
+		)
+		.get_matches();
```

Comment:
Is there a reason for the change in indentation? There's multiple tabs here while the rest of the repo uses four spaces.

Reply:
Oops, my fault. I'll fix it